package.json

```diff
@@ -1,18 +1,27 @@
 {
   "name": "@bpmn-io/diagram-js-ui",
   "version": "0.2.2",
+  "type": "module",
   "main": "lib/index.js",
   "module": "lib/index.js",
   "types": "lib/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "types": "./lib/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
   "files": [
     "lib"
   ],
   "sideEffects": false,
   "scripts": {
-    "all": "run-s lint test",
-    "lint": "eslint .",
-    "test": "karma start",
-    "dev": "karma start --auto-watch --no-single-run"
+    "all": "run-s lint test check-types",
+    "check-types": "tsc --noEmit",
```

Comment:
Since in the opening comment you used `noImplicitAny`, let's include that in the test. I will add it.